proto/axelar-cork/v1/proposal.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+package axelar_cork.v1;
+
+import "axelar-cork/v1/cork.proto";
+
+option go_package = "github.com/peggyjv/sommelier/x/axelarcork/types";
+
+// note: current plan is to accept either chain name or chain ID. if both and they dont match, error.
+
+message AddManagedCellarIDsProposal {
+  string      title       = 1;
+  string      description = 2;
+  string      chain       = 3;
+  uint64      id          = 4;
+  CellarIDSet cellar_ids  = 5;
+}
```

Comment:
It seems easy to get the wrong ID, or at least easier than getting the wrong name.

Reply:
You can't make protov3 fields required, you can only enforce where they are used.

The idea here, and all places where you see name and ID together, is you can use whichever one you please. If you provide both, it will check to make sure they point at the same chain